runtime/doc/options.txt

```diff
@@ -1358,13 +1347,12 @@ A jump table for the options with a short description can be found at |Q_op|.
 	(parts of 'cdpath' can be passed to the shell to expand file names).
 
 						*'cedit'*
-'cedit'			string	(Vi default: "", Vim default: CTRL-F)
+'cedit'			string	(Vim default: CTRL-F, Vi default: "")
```

Comment:
The majority of other option headers have the "Vim default" first, so make it so here as well. There are a few other instances of this.